internal/adapters/entrypoints/rest/assets/static/configUtils.js

```diff
@@ -106,6 +107,15 @@ function hasDuplicateConfirmationAmounts(confirmationArray) {
     return uniqueAmounts.size < amounts.length;
 }
 
+const formatCap = (value, unit) => {
+    try {
+        const num = parseFloat(value);
+        return parseFloat(num.toFixed(4)).toString() + ' ' + unit;
+    } catch (e) {
+        return value + ' ' + unit;
```

Comment:
shouldn't you show the error in this case? how can you be sure that value and unit are correct values if the try block failed?